test/models/config_test.rb

```diff
@@ -41,6 +41,25 @@ class ConfigTest < ActiveSupport::TestCase
       assert @bad_config.errors.messages[:config_key].include? "can't be blank"
     end
 
+    it 'should reject lists for singleton values' do
```

Comment:
👨‍🍳

Reply:
are you ok with how this is structured (i.e. test of the functionality itself but not actually checking that all the fields validate in that way)? or flip it around?

Reply:
nah I think this is a good test - taking a pair of representative examples is I think a good enough proveit here.